README.md

```diff
@@ -22,4 +23,18 @@ geojson2csv('/path/to/points.geojson', '/path/to/points.csv', function(err){
 ```bash
 cd test
 mocha .
-```
\ No newline at end of file
+```
+
+**Command Line**
+
+Running from Command Line
+
+1. absolute path:
+	```
+	$ ./node_modules/.bin/geojson2csv inputfile outputfile
+	```
+
+2. with npm
+	```
+	$ num run geojson2csv inputfile outputfile
```

Comment:
Thanks for your quick reaction @tsamaya! Anyhow this should read

    npm run -- geojson2csv inputfile outputfile

as of the second paragraph in https://docs.npmjs.com/cli/run-script. 
Also when prepending four spaces to a line, it is automatically set as formatted code. No need to wrap it additionally in ```.

Reply:
ooh thanks, I was too fast committing

not sure for the `--` as the `inputfile` and  `outputfile` are not option parameters (like --help or -H)

will update later